metadata-models/src/main/pegasus/com/linkedin/ml/metadata/BaseData.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.DatasetUrn
+
+/**
+ * BaseData record
+ */
+record BaseData {
+
+  /**
+   * What datasets were used in the MLModel?
+   */
+  datasets: optional DatasetUrn
```

Comment:
minor comment: `dataset` not `datasets`? There are other references to `datasets` in field description(s) which can be changed to singular form as well.
Also, should `dataset` be a required field here?